packages/page-supersig/src/Supersig/Dashboard/useLoadingDelay.ts

```diff
@@ -0,0 +1,17 @@
+// Copyright 2017-2021 @polkadot/react-hooks authors & contributors
+// SPDX-License-Identifier: Apache-2.0
+import { useEffect, useState } from 'react';
+import { useIsMountedRef } from '../../../../react-hooks/src/useIsMountedRef';
+export function useLoadingDelay (delay = 100): boolean {
```

Comment:
This is already existing in `react-hooks`?

Reply:
i believe you removed useLoadingDelay (may be mistaken) which is why i brought it locally into page-supersig.